cdap-ui/app/cdap/components/Cloud/Profiles/Store/ActionCreator.js

```diff
@@ -153,3 +152,77 @@ export const getProvisionerLabel = (profile, provisioners) => {
   }
   return profile.provisioner.name;
 };
+
+export const extractProfileName = (name = '') => {
+  return name.replace(/(USER|SYSTEM):/g, '');
```

Comment:
seems a bit dangerous. I can't have profile with `USER` or `SYSTEM` in the name?

Reply:
You can't have a profile with `USER:` or `SYSTEM:` in the name, but the backend throws an error message if the name contains `:` anyways